internal/database/conf.go

```diff
@@ -166,16 +157,30 @@ func scanSiteConfig(s dbutil.Scanner) (*SiteConfig, error) {
 	return &c, nil
 }
 
-func (s *confStore) ListSiteConfigs(ctx context.Context, opt SiteConfigListOptions) ([]*SiteConfig, error) {
-	// Ascending order by default.
-	orderByClause := sqlf.Sprintf("ORDER BY id ASC")
-	if opt.OrderByDirection == DescendingOrderByDirection {
-		orderByClause = sqlf.Sprintf("ORDER BY id DESC")
+func (s *confStore) ListSiteConfigs(ctx context.Context, paginationArgs *PaginationArgs) ([]*SiteConfig, error) {
+	where := []*sqlf.Query{sqlf.Sprintf(`type = 'site'`)}
+
+	// This will fetch all site configs.
+	if paginationArgs == nil {
+		query := sqlf.Sprintf(listSiteConfigsFmtStr, sqlf.Sprintf("WHERE (%s)", sqlf.Join(where, "AND")))
+		rows, err := s.Query(ctx, query)
+		return scanSiteConfigs(rows, err)
+	}
+
+	args, err := paginationArgs.SQL()
+	if err != nil {
+		return []*SiteConfig{}, nil
+	}
+
+	if args.Where != nil {
+		where = append(where, args.Where)
 	}
 
-	q := sqlf.Sprintf(listSiteConfigsFmtStr, orderByClause, opt.LimitOffset.SQL())
+	query := sqlf.Sprintf(listSiteConfigsFmtStr, sqlf.Sprintf("WHERE (%s)", sqlf.Join(where, "AND")))
```

Comment:
Why not put the `WHERE (%S)` in `listSiteConfigsFmtStr` and then do this?

```suggestion
	query := sqlf.Sprintf(listSiteConfigsFmtStr, sqlf.Join(where, "AND"))
```

Reply:
Great suggestion. Fixed now. 👌🏽